src/backend/binaryen/utils.ts

```diff
@@ -655,7 +668,8 @@ export namespace FunctionalFuncs {
 
         if (
             typeKind === ValueTypeKind.ANY ||
-            typeKind === ValueTypeKind.UNION
+            typeKind === ValueTypeKind.UNION ||
+            typeKind === ValueTypeKind.TYPE_PARAMETER
```

Comment:
Would this be conflict with generic feature?

Reply:
No, for Type_Parameter kind, we always regard it as `any` type, if `<T>` has been specialized, then its type is not Type_Parameter anymore.